pixelmatch/contrib/PIL.py

```diff
@@ -0,0 +1,93 @@
+"""Functions to facilitate direct comparison of PIL.Image instances"""
+from typing import Optional, List, Tuple
+
+from PIL.Image import Image # type: ignore
```

Comment:
Have you considered updating `mypy.ini` instead? https://github.com/whtsky/pixelmatch-py/blob/7ed84e4d57fdc70bb24df5a98102b8c009c933bb/mypy.ini

Reply:
I originally went this route because it's more explicit, but this method is cleaner and it's a small thing anyways.